pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -656,9 +714,10 @@ func waitUntilDNSReady(zedagentCtx *zedagentContext, stillRunning *time.Ticker)
 		case change := <-zedagentCtx.subLocationInfo.MsgChan():
 			zedagentCtx.subLocationInfo.ProcessChange(change)
 
-		case change := <-zedagentCtx.deferredChan:
+		case change := <-zedcloudCtx.DeferredEventCtx.Ticker.C:
 			start := time.Now()
-			zedcloud.HandleDeferred(zedcloudCtx, change, 100*time.Millisecond, false)
+			zedcloudCtx.DeferredEventCtx.HandleDeferred(
+				change, 100*time.Millisecond, false)
 			zedagentCtx.ps.CheckMaxTimeTopic(agentName, "deferredChan", start,
```

Comment:
Want to rename the deferredChan to deferredEventCtx?

Reply:
Done.